backend/src/test/java/de/capstonemarc/backend/baby/IntegrationTest.java

```diff
@@ -20,13 +22,21 @@ class IntegrationTest {
     private ObjectMapper objectMapper;
 
     @Test
+    @DirtiesContext
     void addBaby() throws Exception {
         mockMvc.perform(MockMvcRequestBuilders.post("/api/baby")
                 .contentType(APPLICATION_JSON)
-                .content(objectMapper.writeValueAsString(new NewBaby("Baby")))
-        )
+                .content(objectMapper.writeValueAsString(new NewBaby("Baby", "01.01.2000", "3.5", "76", "w"))))
                 .andExpect(status().is(201));
 
 
     }
+
+    @Test
+    @DirtiesContext
+    void getAllBabies() throws Exception {
+        mockMvc.perform(MockMvcRequestBuilders.get("/api/baby"))
```

Comment:
Die REST-Pfade, die sich auf mehrere Elemente beziehen, sollten im Plural formuliert sein.
```suggestion
        mockMvc.perform(MockMvcRequestBuilders.get("/api/babies"))
```